tests/test_deprecated.py

```diff
@@ -34,9 +34,14 @@ def test_tbd_remove_in_v0_8_0_trainer():
         'max_nb_epochs': 'max_epochs',
         'min_nb_epochs': 'min_epochs',
         'nb_sanity_val_steps': 'num_sanity_val_steps',
+        'fast_dev_run': 'unit_test',
```

Comment:
this is a deprecated test for v0.8 but you change is till v0.9 so move it to correct func

Reply:
I have done the requested changes. Kindly review whenever possible.